doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -7,6 +7,8 @@ To create the experiment directory and workflow when running the SRW Application
 
 There is an extensive list of experiment parameters that a user can set when configuring the experiment. Not all of these parameters need to be set explicitly by the user in ``config.yaml``. If a user does not define a variable in the ``config.yaml`` script, its value in ``config_defaults.yaml`` will be used, or the value will be reset depending on other parameters, such as the platform (``MACHINE``) selected for the experiment. 
 
+When a Jinja expression is used to refer to other values, it will be evaluated by ``uwtools`` as a string. In some cases, it is important that the value is appropriately typed. The ``uwtools`` YAML language supports several pyYAML tags that apply appropriate typing. You can find more information about the Custom Tags in the `uwtools docs <https://uwtools.readthedocs.io/en/main/sections/user_guide/yaml/tags.html>__`. For example, those may be ``!int``, ``!float``, ``!bool``, etc.
```

Comment:
@christinaholtNOAA This link isn't rendering correctly in the compiled docs, not sure why. Otherwise the note looks fine. It would be nice if this could be something like a footnote that could be linked to wherever one of these tags appears in the defaults, but that's more a suggestion for the documentation team.

Can you also modify the default values below to include the tags you have added to config_defaults.yaml? For example, `FCST_LEN_CYCL` and `LONG_FCST_LEN_HRS ` now have an `!int` tag in their default. Or will this file be updated automatically for the release? I actually see

Reply:
Okay, I think I took care of both of these requests. Let me know if there are other things I can update. I stayed away from documenting values that weren't already documented.